clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -68,8 +68,8 @@
  * int anotherConfig = (Integer) configs.get(&quot;config_with_validator&quot;);
  *
  * To validate the full configuration, use:
```

Comment:
Should this line be commented? i.e. 
`// To validate the full configuration, use:`